### ISSUE TITLE
Bug: could not run npx siimple -c -o without changing a file in node_modules

### ISSUE THREAD

**Issue by pepijnmm** [jmjuanes/siimple#192]
### Describe the issue

when I tried to run the setup commands following https://siimple.xyz/getting-started/
I got the following error on "npx siimple -c ./siimple.config.mjs -o ./output.css"
see screenshot.

### To reproduce

for me, it just happened after doing the second step.
my npm is 8.19.2
node v16.13.0

### Screenshots

![image](https://user-images.githubusercontent.com/3217184/194944882-3c2d5e60-49e8-4420-9faf-146a27fba730.png)

### What version of **siimple** are you using?

4.3.0(according to package/package-lock)

### Additional context

I could "fix" it by reading a bit on https://stackoverflow.com/a/70057245/4769837
where I read Imports can only do file:// and data://.
So I changed where the error came from on siimple/cli/index.js from:
```
    return import(configPath).then(rawConfig => rawConfig.default);
```
to:
```
    return import("file://" + configPath).then(rawConfig => rawConfig.default);
```
which fixed it but of course not a great solution.

**Comment by jmjuanes** [comment]
Hello @pepijnmm. 

Instead of concatenating `"file://"` to the configuration path we can use [`url.pathToFileURL`](https://nodejs.org/api/url.html#urlpathtofileurlpath):

```
return import(url.pathToFileURL(configPath)).then(rawConfig => rawConfig.default);
```

As I see in the documentation is the recommended way when using absolute paths with `import` https://nodejs.org/api/esm.html#file-urls.

**Comment by jmjuanes** [comment]
Hello @pepijnmm I have just published `v4.3.1` with the patch for your issue. Now it should work as expected. If the issue persists, please feel free to reopen the issue.

Thanks!!
